cmd/global-heal.go

```diff
@@ -224,7 +224,8 @@ func (er *erasureObjects) healErasureSet(ctx context.Context, buckets []string,
 
 		disks, _ := er.getOnlineDisksWithHealing(false)
 		if len(disks) == 0 {
-			logger.LogIf(ctx, fmt.Errorf("no online disks found to heal the bucket `%s`", bucket))
+			// all disks are healing in this set, assume object healing is done.
+			tracker.bucketDone(bucket)
```

Comment:
I am not sure I understand this, first if all disks are healing, this means we lost data already here, what is the point of doing bucketDone() here ?